src/Components/Components/test/NavigationManagerTest.cs

```diff
@@ -60,24 +62,6 @@ public void Initialize_ThrowsForInvalidBaseRelativePaths(string baseUri, string
             ex.Message);
     }
 
-    [Theory]
-    [InlineData("scheme://host/", "otherscheme://host/")]
-    [InlineData("scheme://host/", "scheme://otherhost/")]
-    [InlineData("scheme://host/path/", "scheme://host/")]
-    public void Uri_ThrowsForInvalidBaseRelativePaths(string baseUri, string absoluteUri)
-    {
-        var navigationManager = new TestNavigationManager(baseUri);
-
-        var ex = Assert.Throws<ArgumentException>(() =>
-        {
-            navigationManager.ToBaseRelativePath(absoluteUri);
-        });
-
-        Assert.Equal(
-            $"The URI '{absoluteUri}' is not contained by the base URI '{baseUri}'.",
-            ex.Message);
-    }
```

Comment:
Should this test be removed, or is this for some kind of work-in-progress reason? If it's being removed for real, could you clarify why? I'm just not sure why it would be affected by this PR :)

Reply:
When going through NavigationManagerTests I noticed that Uri_ThrowsForInvalidBaseRelativePaths is exact same as the one after it ToBaseRelativePath_ThrowsForInvalidBaseRelativePaths. The only difference is the name. I decided to keep ToBaseRelativePath_ThrowsForInvalidBaseRelativePaths since its name is more descriptive.

Reply:
Ah great, well spotted! Thanks for sorting that out.